### PR TITLE
Replace a bad image tag

### PR DESCRIPTION
**Pull request by yiboliu**
### Reasons for making this change
jenkins:latest doesn't work anymore. Even though it exists [here](https://hub.docker.com/_/jenkins), `docker pull jenkins:latest` or `docker pull alpine` works neither locally nor on the dev server. I replaced it with a specific version tag according to this [article](https://vsupalov.com/docker-latest-tag/). 

### Checklist

* [x] I've added a screenshot of the changes, if this is a frontend change
* [x] I've added and/or updated tests, if this is a backend change
* [x] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [x] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

**Comment by nelson-liu** [comment]
The Docker image registry says that the `jenkins` image is deprecated, and that you should use https://hub.docker.com/r/jenkins/jenkins instead (`jenkins/jenkins`) ---maybe we should switch?

File: tests/stress/stress_test.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ class StressTestRunner:
         'couchbase:latest',
         'golang:latest',
         'iwane/numpy-matplotlib',
-        'jenkins:latest',
+        'jenkins:jenkins',
```
Comment (teetone) [inline tests/stress/stress_test.py]: Should it be `jenkins/jenkins:latest`?

**Review by teetone — CHANGES_REQUESTED** [review]

File: tests/stress/stress_test.py (hunk 1 above)
Comment (yiboliu) [reply tests/stress/stress_test.py]: the default tag is `latest`. So it will automatically use `latest`

Comment (teetone) [reply tests/stress/stress_test.py]: No, I meant it should be `jenkins/jenkins`, right?

**Review by teetone — APPROVED** [review]

**Pull request merged**